test/parallel/test-assert.js

```diff
@@ -55,6 +55,52 @@ assert.throws(() => a.ok(false), a.AssertionError, 'ok(false)');
   assert.ok(threw, 'Error: ok(false)');
 }
 
+// Throw message if the message is instanceof Error.
+{
+  let threw = false;
+  const context = vm.createContext();
+  try {
+    const errorConstructor = vm.runInContext('Error', context);
+    const customError = new errorConstructor('ok(false)');
+    assert.ok(false, customError.message);
+  } catch (e) {
+    threw = true;
+    assert.ok(e instanceof Error);
+  }
+  assert.ok(threw, 'Error: ok(false)');
```

Comment:
This specific case seems to test the second argument being a string. That is something that we should already have tests for and I guess we should just remove it?